### PR TITLE
Updated opportunities/_styles.less 

### LINKED ISSUE #201: Switch css changes on Beacon to .less
Oops.
- th class
- a class for table

### PR DESCRIPTION
**Pull request by msknee**
#### What Changed
- Fixed #201 
- Table headings, table links, and body link style changes
#### What's Needed

N/A
#### Screenshots

Table links are bolded, and table heading text is smaller and lighter

screen shot 2015-08-12 at 2 30 07 pm

Links on Beacon are now bolded everywhere except the navbar:

screen shot 2015-08-12 at 2 30 20 pm

### CODE REVIEW

File: purchasing/templates/wexplorer/search.html [hunk 1]
```diff
@@ -46,10 +46,7 @@ <h3 class="text-muted">
       <table class="table table-striped" id="js-sort-results">
         <thead>
           <tr>
-            {% if current_user.is_anonymous() == False %}
-            <th>Star</th>
-            {% endif %}
-            <th class="js-sortable-th">Contract Discription</th>
+            <th class="js-sortable-th">Contract Description</th>
```
Comment (bsmithgall) [inline purchasing/templates/wexplorer/search.html:49]: Can you revert this change so that it doesn't cause a merge conflict with #212

File: purchasing/static/less/opportunities/_styles.less [hunk 2]
```diff
@@ -1,3 +1,12 @@
+
+a {
```
Comment (bsmithgall) [inline purchasing/static/less/opportunities/_styles.less:2]: Do you mean to change links globally? This will change all of them across all of Beacon.

Comment (msknee) [inline purchasing/static/less/opportunities/_styles.less:2]: Yup.

**Comment by hamhands** [comment]
LGTM :+1:

**Comment by hamhands** [comment]
Clear to merge @bsmithgall?
